features/select_projects_metrics_based_on_preference.feature

```diff
@@ -38,24 +38,24 @@ Scenario: select preferred projects
 
 Scenario: select preferred metrics
 	When I am on the projects page
-	Then I should see "2.9" within "#LocalSupport_code_climate_metric"
-	And I should see "0.6" within "#WebsiteOne_github_metric"
+	Then I should see "2.9" within "#project_1_code_climate_metric"
```

Comment:
that was because when I was developing this feature, there's a project named "Alzheimer's XXXX" which caused the css selection to fail because of the `'` in the middle of its name